setup.py

```diff
@@ -51,6 +51,7 @@ def get_version():
         'setuptools',
         'numpy',
         'iminuit',
+        'matplotlib'
```

Comment:
Well, I thought that at first, but the problem is some modules that (I would guess) are nominally unrelated to plotting do `import matplotlib`. For example [`oneshot`](https://github.com/iminuit/probfit/blob/3b4ecdc42a066bdd7525b1543c3af409536b92a5/probfit/oneshot.py#L4) and [`toy`](https://github.com/iminuit/probfit/blob/3b4ecdc42a066bdd7525b1543c3af409536b92a5/probfit/toy.py#L4).

Because these modules are [imported in the probfit `__init__`](https://github.com/iminuit/probfit/blob/3b4ecdc42a066bdd7525b1543c3af409536b92a5/probfit/__init__.py#L12-L14), then one can't `import probfit` without matplotlib. So, right now I'd say matplotlib is a dependency.